components/pages/explorer/RepoTable.tsx

```diff
@@ -229,19 +235,84 @@ const getTableStyle = (theme: typeof defaultTheme) => css`
   }
 `;
 
+const getDownloadError = (status: number, errText?: string) => {
+  switch (status) {
+    case 400:
+      if (errText === 'No files found.') {
+        return 'The selected sequences do not have files associated with them. Please try again with different sequences.';
+      }
+      return 'Failed to download files for the requested sequences. If the problem persists, contact support for assistance.';
```

Comment:
The `contact support` message may be removed/modified, confirming whether there will be a support account to contact!